src/mbgl/text/glyph_pbf.hpp

```diff
@@ -21,7 +21,7 @@ class SDFGlyph {
     AlphaImage bitmap;
 
     // Glyph metrics
-    GlyphMetrics metrics;
+    optional<GlyphMetrics> metrics;
```

Comment:
I have an alternate fix for glyph-related `operator bool()` prepared as part of #8747, so I'll omit the last commit here and merge.